package.json

```diff
@@ -52,17 +52,17 @@
     "asn1.js": "=4.9.1",
     "asn1.js-rfc3280": "=4.0.0",
     "asn1.js-rfc5280": "=2.0.0",
-    "bitcore-lib": "^0.15.0",
+    "bitcore-lib": "0.14.0",
```

Comment:
Did you try to use the 0.15.0? If yes, what was the issue?

Reply:
There was no issue. I changed it to 0.14.0 so there weren't conflicts with the existing version. Either version works fine. It's actually a peer dependency as I don't believe it is actually used in the code.